magento/magento2
# add isMultipleFiles property to image form element template

Prevent the "Browse to find or drag image here" button to disappear when an image is added in a multiple files image form element.

### Description (*)


The only modification in the code is the addition of the already existing (and utilized) property "isMultipleFiles" into the condition that determine whether the "Browse to find or drag image here" button should be display or not.

In doing so, when the user add an image to the multiple files image form element, the "Browse to find or drag image here" is still there.

This is useful to inform the user that the form element can take multiple images and that these images can be drag and drop, especially when the form element is initially populated with existing images.








### Manual testing scenarios (*)

1. Create an adminhtml UI form with an imageUploader element.
2. Define the "isMultipleFiles" boolean property to true in this element.
3. Check the form, the "Browse to find or drag image here" button disappear after the first image is uploaded. 




### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#39219: add isMultipleFiles property to image form element template
